plugins/scanners/scancode/src/main/kotlin/ScanCodeErrorMappers.kt

```diff
@@ -35,27 +33,20 @@ private val TIMEOUT_ERROR_REGEX = Regex(
             "ERROR: Processing interrupted: timeout after (?<timeout>\\d+) seconds. \\(File: (?<file>.+)\\)"
 )
 
-private fun getInputPath(result: JsonNode): String {
-    val header = result["headers"].single()
-    val input = header["options"]["input"]
-    val path = input.takeUnless { it.isArray } ?: input.single()
-    return path.textValue().let { "$it/" }
-}
-
 /**
  * Map scan errors for all files using messages that contain the relative file path.
  */
-internal fun mapScanErrors(result: JsonNode): List<Issue> {
-    val input = getInputPath(result)
-    return result["files"]?.flatMap { file ->
-        val path = file["path"].textValue().removePrefix(input)
-        file["scan_errors"].map {
+internal fun mapScanErrors(result: ScanCodeResult): List<Issue> {
+    val input = result.headers.single().options.input.single()
+    return result.files.flatMap { file ->
+        val path = file.path.removePrefix(input).removePrefix("/")
```

Comment:
Just wondering, why was `removePrefix("/")` not required before?

Reply:
It was done differently before: `getInputPath()` was ensured to end with a `/`, so `removePrefix(input)` also removed that trailing slash.